share/gpodder/ui/gtk/gpodderpreferences.ui

```diff
@@ -1015,82 +1015,92 @@
                             <property name="orientation">vertical</property>
                             <property name="spacing">6</property>
                             <child>
-                              <!-- n-columns=2 n-rows=3 -->
-                              <object class="GtkGrid" id="table_video">
+                              <object class="GtkFlowBox">
                                 <property name="visible">True</property>
                                 <property name="can-focus">False</property>
-                                <property name="row-spacing">6</property>
-                                <property name="column-spacing">12</property>
+                                <property name="min-children-per-line">1</property>
+                                <property name="max-children-per-line">2</property>
+                                <property name="selection-mode">none</property>
                                 <child>
-                                  <object class="GtkLabel" id="label_preferred_youtube_format">
+                                  <object class="GtkFlowBoxChild">
                                     <property name="visible">True</property>
                                     <property name="can-focus">False</property>
-                                    <property name="label" translatable="yes">Preferred YouTube format:</property>
-                                    <property name="xalign">0</property>
+                                    <child>
+                                      <object class="GtkLabel" id="label_preferred_youtube_format">
+                                        <property name="visible">True</property>
+                                        <property name="can-focus">False</property>
+                                        <property name="label" translatable="yes">Preferred YouTube format:</property>
+                                        <property name="xalign">0</property>
+                                      </object>
+                                    </child>
                                   </object>
-                                  <packing>
-                                    <property name="left-attach">0</property>
-                                    <property name="top-attach">0</property>
-                                  </packing>
                                 </child>
                                 <child>
-                                  <object class="GtkComboBox" id="combobox_preferred_youtube_format">
+                                  <object class="GtkFlowBoxChild">
                                     <property name="visible">True</property>
                                     <property name="can-focus">False</property>
-                                    <property name="hexpand">True</property>
```

Comment:
On wide dialogs, the combo boxes can be far from the labels, making it harder to match them up. Is it possible to only stretch the combo boxes and not the labels?

The two centered widgets on the updating page have the same problem. Is it possible to have them immediately after the labels and wrap if not enough space?

Can the label+sliders on updating and cleanup pages, and the label+combo boxes on devices page also be made to flow?

Reply:
I tried various options in the FlowBox and its children, but it looks that extra space is always allocated evenly to both children in a row, so there will be a gap between the label and the combobox.

The combobox and spin button in the Updating page can be put into flowboxes.

The sliders do not work that well with FlowBoxes, the width is again allocated evenly to the label and slider, which results in a gap and wasted space. The 'Update interval' slider could be converted to a ComboBox though.

Reply:
Changing the auto-download combobox halign from center to fill would stretch it like all other comboboxes.

Can the labels+widgets on devices page be put in flow boxes?

Reply:
The Devices page label+widgets are now FlowBoxed, and the auto-download combobox set to fill.